src/Server.php

```diff
@@ -349,6 +375,12 @@ public function getCacheWithFileExtensions()
      */
     public function getCachePath($path, array $params = [])
     {
+        $customCallable = $this->getCachePathCallable();
+        if (null !== $customCallable) {
+            $boundCallable = \Closure::bind($customCallable, $this, self::class);
```

Comment:
Since you are binding current object's context can you please add a test for it too. For e.g. calling `$this->getSourcePath()` inside the callable.

Reply:
@mlazze Ping